src/Topics/StandardModernPrecision/OneDiamondResponses.hs

```diff
@@ -0,0 +1,271 @@
+module Topics.StandardModernPrecision.OneDiamondResponses(topic) where
+
+import Output(output, Punct(..))
+import Topic(Topic(..), wrap, Situations)
+import Auction(withholdBid, {-forbid, makePass,-} maxSuitLength, minSuitLength, {-suitLength,-}
+               {-Action, balancedHand,-} pointRange{-, SuitLengthComparator(..), compareSuitLength-}, displayLastCall)
+import Situation(situation, (<~))
+--import CommonBids(cannotPreempt)
+import qualified Terminology as T
+import Topics.StandardModernPrecision.BasicBids(oppsPass, b1D, smpWrapN)
+import qualified Topics.StandardModernPrecision.Bids1D as B
+
+
+oneMajor :: Situations
+oneMajor = let
+    sit bid = let
+        action = do
+            b1D
+            oppsPass
+            withholdBid bid
+        explanation fmt =
+            "Let's start with a natural " ++ displayLastCall fmt bid ++ "\
+           \ bid, and see where things go from there."
+      in
+        situation "1M" action bid explanation
+  in
+    smpWrapN $ return sit <~ [B.b1D1H, B.b1D1S]
+
+
+twoMinor6M :: Situations
+twoMinor6M = let
```

Comment:
How about a `twoMinorWithoutMajor` too? or maybe there should just be a `twoMinor` Situation, and this one goes into an upcoming file containing the rest of the 1D-2m convention?

Reply:
There are always more situations to create (what if you've got a single minor but are game forcing? What if you've got a 4-card major, 6-card minor, and are only invitational, what about 6-5 and invitational?), but this is certainly a start. I added in two more situations in which you respond with 2m. Don't let the "perfect" get in the way of the "good enough for now."